.github/workflows/pr.yml

```
@@ -14,13 +14,9 @@ jobs:
-        with:
-          registry-url: https://registry.npmjs.org/
-          scope: '@kingstinct'
-
       - run: bun install
 
       - run: bun run prepublishOnly
+        working-directory: packages/react-native-device-activity
       
-      - run: npx pkg-pr-new publish
\ No newline at end of file
+      - run: bunx pkg-pr-new@latest publish './packages/*'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)